### PR TITLE
added simplified github actions

### PR DESCRIPTION
**Pull request by bhushan**
1. Added PHPUnit GitHub action
2. Added PHP CS Fixer GitHub action
3. Removed not needed part from php_cs.dist

### CODE REVIEW

**Comment by bhushan** [comment]
What you did @Wulfheart .. hehe 

It's the third time I am creating PR.. we are now missing phpunit.xml.dist into the dev branch. 

Checking what's going on.. Will add things in the same PR.

File: .github/workflows/php-unit.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, sqlite, pdo_sqlite, bcmath, soap, intl, gd, exif, iconv, imagick
+          coverage: none
+
+      - name: Install Dependencies
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-suggest --no-progress --prefer-dist
```
Comment (Gummibeer) [inline .github/workflows/php-unit.yml]: It should use ansi output as GitHub actions support it.

File: .github/workflows/php-cs-fixer.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Check and fix code styling with php-cs-fixer
+
+on:
+  pull_request:
+  push:
```
Comment (Gummibeer) [inline .github/workflows/php-cs-fixer.yml]: This doesn't makes sense. If any filter it should do on every push but only on PR to the main branches. But I wouldn't filter at all.

File: .github/workflows/php-unit.yml (hunk 1 above)
Comment (bhushan) [reply .github/workflows/php-unit.yml]: Ok changing it..

File: .github/workflows/php-cs-fixer.yml (hunk 2 above)
Comment (bhushan) [reply .github/workflows/php-cs-fixer.yml]: on: [ pull_request, push ]


@Gummibeer  is this fine ?

**Comment by bhushan** [comment]
@Gummibeer You can review PR again.. feedback changes are added into the `feedback changes` commit.

**Review by Wulfheart — APPROVED** [review]

**Comment by Wulfheart** [comment]
Thanks for your contribution, bhushan.

Right now I'm checking it out.

**Pull request merged**